shared/package.json

```diff
@@ -0,0 +1,37 @@
+{
```

Comment:
i'm worried about this and it being out of date. if we mark the values as "0.0.0" does that still make eslint happy?